pkg/controller/audit/actuator.go

```diff
@@ -369,6 +369,16 @@ func seedObjects(auditConfig *v1alpha1.AuditConfig, cluster *extensions.Cluster,
 		return nil, fmt.Errorf("unable to generate webhook kubeconfig: %w", err)
 	}
 
+	pauseInputOnOverLimit := "off"
+	if auditConfig.WebhookMode == v1alpha1.AuditWebhookModeBlockingStrict {
```

Comment:
In the mutating webhook we have a [fallback to blocking-strict](https://github.com/stackitcloud/gardener-extension-audit/blob/dcb227487441f6692e9c06bc72b8eb16c88fba96/pkg/webhook/kapiserver/ensurer.go#L74-L80) in case `webhookMode` is set to empty string. Unfortunately, this is a possible setting at the moment due to a failing admission controller (missing defaulting or validation). :see_no_evil: 

Can we maybe just put a similar switch statement here to achieve consistent behavior?

Reply:
I've added a corresponding switch statement.